site/docs/status.md

```diff
@@ -0,0 +1,362 @@
+| ORC     | Y    | N         | N    | N  |
+| Puffin  | Y    | N         | N    | N  |
+
+## File IO
+
+| Storage              | Java | PyIceberg | Rust | Go |
+|----------------------|------|-----------|------|----|
+| Local Filesystem     | Y    | Y         | Y    | Y  |
+| Hadoop Filesystem    | Y    | Y         | Y    | Y  |
+| Aws S3               | Y    | Y         | Y    | Y  |
+| Google Cloud Storage | Y    | Y         | Y    | Y  |
+| Memory Fs            | Y    | Y         | Y    | Y  |
+
+## Table Update Operations
+
+### Table Spec V1
+
+| Operation                   | Java | PyIceberg | Rust | Go |
+|-----------------------------|------|-----------|------|----|
+| Update schema               | Y    | N         | N    | N  |
+| Update partition spec       | Y    | N         | N    | N  |
+| Update table properties     | Y    | Y         | Y    | N  |
+| Replace sort order          | Y    | N         | N    | N  |
+| Update table location       | Y    | N         | N    | N  |
+| Append data files           | Y    | N         | N    | N  |
+| Rewrite files               | Y    | N         | N    | N  |
+| Rewrite manifests           | Y    | N         | N    | N  |
+| Overwrite files             | Y    | N         | N    | N  |
+| Row delta                   | Y    | N         | N    | N  |
+| Delete files                | Y    | N         | N    | N  |
+| Update statistics           | Y    | N         | N    | N  |
+| Update partition statistics | Y    | N         | N    | N  |
+| Expire snapshots            | Y    | N         | N    | N  |
+| Manage snapshots            | Y    | N         | N    | N  |
+
+### Table Spec V2
+
+| Operation                   | Java | PyIceberg | Rust | Go |
+|-----------------------------|------|-----------|------|----|
+| Update schema               | Y    | N         | N    | N  |
+| Update partition spec       | Y    | N         | N    | N  |
+| Update table properties     | Y    | Y         | Y    | N  |
+| Replace sort order          | Y    | N         | N    | N  |
+| Update table location       | Y    | N         | N    | N  |
+| Append data files           | Y    | N         | N    | N  |
+| Rewrite files               | Y    | N         | N    | N  |
+| Rewrite manifests           | Y    | N         | N    | N  |
+| Overwrite files             | Y    | N         | N    | N  |
+| Row delta                   | Y    | N         | N    | N  |
+| Delete files                | Y    | N         | N    | N  |
+| Update statistics           | Y    | N         | N    | N  |
+| Update partition statistics | Y    | N         | N    | N  |
+| Expire snapshots            | Y    | N         | N    | N  |
+| Manage snapshots            | Y    | N         | N    | N  |
+
+## Table Read Operations
+
+### Table Spec V1
+
+| Operation           | Java | PyIceberg | Rust | Go |
+|---------------------|------|-----------|------|----|
+| Plan with data file | Y    | N         | Y    | N  |
```

Comment:
Both Go and Pyiceberg support planning and reading data files for v1

Reply:
```suggestion
| Plan with data file | Y    | Y         | Y    | Y  |
```